src/components/organisms/ProfileModal/ProfilePopoverContent.scss

```diff
@@ -2,7 +2,20 @@
 
 #profile-popover {
   &.popover {
-    height: unset;
+    &::-webkit-scrollbar-track {
```

Comment:
So, *BLIP* for _-moz-_ or _-ms-_ users, right?

Reply:
When I first started we weren't supposed to support multiple browsers and the users should always use Chrome, maybe something changed. @thecadams Is the project running on Chrome only?

Reply:
I know its only Chrome (for now), its not a required change to include different browser prefixes, its just something to keep in mind long term

Reply:
Chrome only; tolerable on Safari; this is fine to include.